project/yang2829/README.rst

```diff
@@ -0,0 +1,63 @@
+Voronoi Diagram
+===============
+
+Description
+------------
+GitHub: https://github.com/yang2829/Voronoi-Diagram
+
+Problem to Solve
+----------------
+The Voronoi diagram is a decomposition of a region that any point in a sub-region is closest to the site of the sub-region.
+
+.. image:: https://upload.wikimedia.org/wikipedia/commons/5/54/Euclidean_Voronoi_diagram.svg
+   :width: 40%
+
+Voronoi diagrams are widely used in fields such as Geographic Information Systems (GIS) and Computer Graphics.
+This project is limited to 2-dimensional space. It may use Fortune's algorithm to solve it.
+
+Prospective Users
+-----------------
+This system will serve as a useful tool for users needing to analyze spatial relationships and optimize region-based solutions.
+
+System Architecture
+-------------------
+Input:
+
+* A set of 2-dimensional points.
```

Comment:
A "set" sometimes means an unordered collection.  Is it your intention?

Reply:
Yes. I will sort it by my self if needed.

Reply:
It is OK to sort or not as long as you know the characteristics of the data structure you use.